### PR TITLE
Ballots cast per contest

### PR DESCRIPTION
**Pull request by benadida**
Tabulates all yes/no votes correctly with new format.

### CODE REVIEW

File: client/src/components/ContestTally.tsx [hunk 1]
```diff
@@ -37,9 +46,23 @@ const ContestTally = ({ election, electionTally }: Props) => {
...
 
+        const {
+          ballots,
+          overvotes,
+          undervotes,
+        }: ContestTallyMeta = contestTallyMeta[contest.id]!
         return (
           <Contest key={`div-${contest.id}`}>
             <Prose maxWidth={false}>
+              {ballots && (
```
Comment (beausmith) [inline client/src/components/ContestTally.tsx:57]: `!!ballots`

File: client/src/lib/votecounting.ts [hunk 2]
```diff
@@ -151,7 +152,14 @@ export function tallyVotesByContest({
 
       if (contest.type === 'yesno') {
         const optionTally = find(tallies, (optionTally) => {
-          return optionTally.option === selected
+          // console.log({
+          //   contest,
+          //   option: (optionTally as YesNoContestOptionTally).option[0],
+          //   selected: selected[0],
+          // })
```
Comment (beausmith) [inline client/src/lib/votecounting.ts:159]: Delete comment

File: client/src/lib/usbstick.test.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import fakeKiosk from '../../test/helpers/fakeKiosk'
+import {
+  isAvailable,
+  isPresent,
+  isMounted,
+  doMount,
+  doUnmount,
+} from './usbstick'
```
Comment (beausmith) [inline client/src/lib/usbstick.test.ts:8]: Why is there USB stuff in this PR?

File: client/src/lib/votecounting.ts [hunk 4]
```diff
@@ -166,7 +174,7 @@ export function tallyVotesByContest({
       }
     })
 
-    contestTallies.push({ contest, tallies })
+    contestTallies.push({ contest, tallies, ballotsCastWithContest: 1 })
```
Comment (beausmith) [inline client/src/lib/votecounting.ts:177]: Remove `, ballots Cast With Contest: 1`

File: client/src/config/types.ts [hunk 5]
```diff
@@ -62,6 +67,7 @@ export interface ContestOptionTally {
 export interface ContestTally {
   contest: Contest
   tallies: ContestOptionTally[]
+  ballotsCastWithContest: number
```
Comment (beausmith) [inline client/src/config/types.ts:70]: Remove this line. It is not used.

File: client/src/lib/usbstick.test.ts (hunk 3 above)
Comment (benadida) [reply client/src/lib/usbstick.test.ts:8]: as per comment: because I needed to add some coverage and this was something that needed it.

**Pull request merged**